user_test.go

```diff
@@ -19,6 +19,45 @@ func TestGetUserInfo(t *testing.T) {
 	require.Nil(t, err)
 	assert.Equal(t, PassageUserID, user.ID)
 }
+
+func TestGetUserInfoByIdentifier(t *testing.T) {
+	psg, err := passage.New(PassageAppID, &passage.Config{
+		APIKey: PassageApiKey,
+	})
+	require.Nil(t, err)
+
+	createUserBody := passage.CreateUserBody{
+		Email: RandomEmail,
+	}
+
+	user, err := psg.CreateUser(createUserBody)
+	require.Nil(t, err)
+	assert.Equal(t, RandomEmail, user.Email)
+
+	userByIdentifier, err := psg.GetUserByIdentifier(RandomEmail)
```

Comment:
This test is only passing now because `RandomEmail` only generates lowercase letters. https://github.com/passageidentity/passage-go/blob/e4363a976dbcef6305a80fc5079e8cee391aae0f/passage_test.go#L29-L30

Another test case that searches for the mixed or uppercase counterpart would be good to cover the case-insensitivity.

Reply:
created specific all uppercase test